### PR TITLE
Implement new deployment script

### PR DESCRIPTION
**Pull request by kermankohli**

### CODE REVIEW

File: deployments/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Getting Started
+
+Before getting started, make sure you have the following `env` variables set:
+
+```
+// Private key to deploy with
+DEPLOYMENT_PRIVATE_KEY="0xA..."
+
+// The network name set here will use constants from `network-constants.ts`
+DEPLOYMENT_NETWORK_NAME="development"
```
Comment (felix2feng) [inline deployments/README.md]: Let's detail which networks you can deploy to. `kovan`, `main` etc.

File: deployments/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+// Network id will indicate which chain to run on (testnet, mainnet etc)
+DEPLOYMENT_NETWORK_ID=50;
```
Comment (felix2feng) [inline deployments/README.md]: Which network corresponds to what? Would be helpful

File: .gitignore [hunk 3]
```diff
@@ -29,3 +29,6 @@ yarn-error.log*
 
 # Snapshot
 blockchain/
+
+# Ouputs
+outputs.ts
```
Comment (felix2feng) [inline .gitignore]: Do you mean deployments/outputs.ts?

Comment (kermankohli) [reply .gitignore]: Not sure if the path makes a difference but added it for granularity's sake

**Pull request merged**